lib/fluent/plugin/in_tail.rb

```diff
@@ -107,6 +109,8 @@ def initialize
     config_param :path_timezone, :string, default: nil
     desc 'Follow inodes instead of following file names. Guarantees more stable delivery and allows to use * in path pattern with rotating files'
     config_param :follow_inodes, :bool, default: false
+    desc 'Specify max size of thread pool'
+    config_param :max_thread_pool_size, :integer, default: 1
```

Comment:
We should describe that this parameter is for the log throttling feature.

Reply:
After removing thread pool feature, we need to remove this needless parameter.